src/components/Table/TransactionTable.js

```diff
@@ -0,0 +1,211 @@
+import React, { PureComponent } from 'react';
+import propTypes from 'prop-types';
+import moment from 'moment';
+import assert from 'bsert';
+import { Table } from '../index';
+import { connectTheme } from '../../utils';
+
+import { TxManager, TxManagerOptions } from '@bpanel/bpanel-utils';
+
+import ExpandedDataRow from './ExpandedDataRow';
+
```

Comment:
Where should the custom footer live? Since this is a transaction table, I think that its ok for the custom footer to live here, maybe there is another directory, `Table/TransactionTable/{TransactionTable,LinkFooter}.js`

Reply:
I don’t think we need a dedicated component for this. If it takes children then they just have to be rendered after the rest of the data. This lets people add whatever they want. The neat thing about this too is someone could create a totally arbitrary expanded row where there’s no data and just the children that renders something totally separate. The transaction table may want to render a link, but some other table may want to add something totally separate.